frontstage/views/register/enter_account_details.py

```diff
@@ -31,8 +32,9 @@ def register_enter_your_details():
     iac_controller.validate_enrolment_code(enrolment_code)
 
     if request.method == "POST" and form.validate():
-        logger.info("Attempting to create account")
         email_address = form.email_address.data
+        logger.info("Attempting to create account", email=obfuscate_email(email_address))
```

Comment:
Do we need this log line to happen twice? One with the iac and one without? (Line 77 of party_controller.py)

Reply:
Good point, removed